src/lib/functions/server.mjs

```diff
@@ -41,6 +41,13 @@ const buildClientContext = function (headers) {
   }
 }
 
+const hasBody = (req) =>
+  // copied from is-type package
+  (req.header('transfer-encoding') !== undefined || !Number.isNaN(req.header('content-length'))) &&
```

Comment:
Why're specifically checking for the `NaN` value here?

In the `type-is` package the global isNaN function is used which behaves differently from the `Number.isNaN` function since the global isNaN is true for any non-numeric value (strings, undefined, null etc.) while the `Number.isNaN` isn't.


Ref: https://github.com/jshttp/type-is/blob/7d19b7aab1ad671f59ba157ae0640cd4b1302ca5/index.js#L92-L95

Reply:
Interesting, I wasn't aware of that and I  just changed it because some eslint rule recommended it.

change to the global version